packages/client/hmi-client/src/components/widgets/tera-slider.vue

```diff
@@ -64,6 +64,10 @@ const sidePanelTabStyle = computed(
 </script>
 
 <style scoped>
+.slider {
+	z-index: 1;
+}
```

Comment:
Please handle z-index within the parent component that's using `tera-slider.vue`.
And when doing so add `isolation: isolate;` to make sure that those z-indexes are not conflicted with other components.
Take a look at `App.vue` for how to implement it.